test/Table_Tests/src/IO/Excel_Spec.enso

```diff
@@ -640,15 +640,15 @@ spec_write suite_builder suffix test_sheet_name =
             parent = enso_project.data / "transient" / "nonexistent"
             parent.exists.should_be_false
 
-            f = parent / "foo."+suffix
+            f = parent / ("foo."+suffix)
```

Comment:
This one is unfortunate. It has nothing to do with arithmetic and looks really bad surrounded by `(` and `)`.

Reply:
Eventually we will have string interpolation, right? The parser supports it.

Reply:
> Eventually we will have string interpolation, right? The parser supports it.

Interesting. I didn't know we are supposed to support string interpolation. Where's the spec? What's the syntax? What `Tree` will one get?

Both `main x = 'x=$x'` and `main x = "x=$x" resulted in a simple text literal on the IR level...`

Reply:
https://github.com/enso-org/design/blob/wip/wd/enso-spec/epics/enso-spec-1.0/04.%20Expressions.md#interpolated-text-literals
The elements returned by `getElements` can return a `TextElement.Splice`, but we flatten all of the elements into a string in TreeToIr to fit the current IR representation.
We also flatten in the GUI. When the backend supports splices, in the frontend we would probably want to display the spliced expression with widgets, but we would need to be careful that it is clear that it is part of the string expression.